thirdeye/thirdeye-pinot/src/main/java/org/apache/pinot/thirdeye/detection/validators/DatasetConfigValidator.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.pinot.thirdeye.detection.validators;
+
+import java.util.Map;
+
+import com.google.common.base.Preconditions;
+import org.apache.pinot.thirdeye.datalayer.dto.DatasetConfigDTO;
+
+public class DatasetConfigValidator implements ConfigValidator<DatasetConfigDTO> {
+  private static final String DEFAULT_DATASET_NAME = "online_dataset";
+
+  @Override
+  public void validateConfig(DatasetConfigDTO config) throws IllegalArgumentException {
+    Preconditions.checkArgument(config.getName().startsWith(DEFAULT_DATASET_NAME));
```

Comment:
Why do we want to check this prefix? It looks like that this prefix is added by our code and it is guaranteed to happen, right? If we just need to validate the prefix, we should remove this class.

Reply:
Yes, you are right! I've removed both this class and metricConfigValidator. The initial purpose of those two classes is to provide a separate class to validate the format of the user input YAML files. Currently, since this endpoint does not support rich customized options, there is not too much to validate so I left them as blank. I deleted those two validators and we could add them back if it's needed in the future.